README.md

```diff
@@ -2,32 +2,32 @@ Rancher Harvester
 ========
 [![Build Status](https://drone-publish.rancher.io/api/badges/rancher/harvester/status.svg)](https://drone-publish.rancher.io/rancher/harvester)
 
-Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes.
+Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes. It is an on-prem cloud computing virtualization platform.
 
 ## Overview
 Harvester makes it easy to get HCI up and running on the bare metal servers. Here are some notable features of the Harvester that currently provided:
 1. VM lifecycle management includes SSH-Key injection, Cloud-init and, graphic and serial port console
 1. Build-in storage management based on [Longhorn](https://longhorn.io/)
 1. Networking management with both overlay management networks and VLAN
-1. Build-in image repository
+1. Built-in image repository
 1. Virtual Machine templates
 
 ## Hardware Requirements
-To get the Harvester server up and running the following minimum hardware requirements are required:
+To get the Harvester server up and running, the following minimum hardware requirements are required:
 
 | Type | Minimum Requirements |
 |:---|:---|
-| CPU | 4C is required, a minimum of 16C is preferred |
-| Memory | 8G is required, a minimum of 32G is preferred |
-| Disk |  at least 120G is required, 500GB or above is preferred |
-| Network Card | at least 1 Gigabit Ethernet port, for example, Ethernet 1GbE, RJ45  |
-| Network Switch | Equipped with at least 1 Gigabit switch, 10 Gigabit switch is recommended(Will need to enabling trunking of port for VLAN support) |
+| CPU | 4 C is required, a minimum of 16 C is preferred |
+| Memory | 8 GB is required, a minimum of 32 GB is preferred |
+| Disk |  at least 120 GB is required, 500 GB or above is preferred |
+| Network Card | at least 1 GB Ethernet port, for example, Ethernet 1GbE, RJ45  |
```

Comment:
It's not a grammar issue. But we should raise the minimal requirement to 10Gbps instead. I will add a separate commit for that.